src/main/java/com/cloudbees/jenkins/support/SupportAction.java

```diff
@@ -146,8 +152,10 @@ public void doDownload(StaplerRequest req, StaplerResponse rsp) throws ServletEx
         if (supportPlugin != null) {
             SupportProvider supportProvider = supportPlugin.getSupportProvider();
             if (supportProvider != null) {
+                SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd_HH.mm.ss");
```

Comment:
Creating a new `SimpleDateFormat` can be an expensive operation especially every time the `doDownload` link is clicked. I would recommend using a `static final` field since the format for the date/time never changes.